server/channels/api4/remote_cluster.go

```diff
@@ -21,7 +21,7 @@ func (api *API) InitRemoteCluster() {
 	api.BaseRoutes.RemoteCluster.Handle("/msg", api.RemoteClusterTokenRequired(remoteClusterAcceptMessage)).Methods("POST")
 	api.BaseRoutes.RemoteCluster.Handle("/confirm_invite", api.RemoteClusterTokenRequired(remoteClusterConfirmInvite)).Methods("POST")
 	api.BaseRoutes.RemoteCluster.Handle("/upload/{upload_id:[A-Za-z0-9]+}", api.RemoteClusterTokenRequired(uploadRemoteData)).Methods("POST")
```

Comment:
@streamer45 can you help me understand what this API is used for? Its receiving a file (as its checking if file uploads are enabled or not), but I'm understand what's this for. Can end users call this API?

Reply:
@harshilsharma63 I believe this would be used by our Shared Channels implementation to automatically sync attachments between connected MM installations.

Reply:
Got it adding now.

Reply:
Done.